python/tvm/driver/tvmc/composite_target.py

```diff
@@ -29,6 +29,7 @@
 from tvm.relay.op.contrib.bnns import partition_for_bnns
 from tvm.relay.op.contrib.vitis_ai import partition_for_vitis_ai
 from tvm.relay.op.contrib.clml import partition_for_clml
+from tvm.relay.op.contrib import adreno
```

Comment:
What happens here is TVM is compiled without OpenCL support, which is the case for, example, tlcpack?

Reference: https://github.com/tlc-pack/tlcpack/blob/d3393f530331f28494b71dc68f60477e6b6dc9de/wheel/build_wheel_manylinux.sh#L88-L103

Reply:
Shouldn't be an issue. ```tvm.relay.op.contrib.adreno``` is just a ```ToMixedPrecision``` invocation and independent of any framework or target.